### PR TITLE
Intake rollers

### PR DESCRIPTION
**Pull request by nonchalantnia**

### CODE REVIEW

File: 2025_PreseasonSwerve/src/main/java/frc/robot/subsystems/intake/rollers/RollersConstants.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems.intake.rollers;
+
+public class RollersConstants {
+
+    public static final int rollerMotorID = 0;
+    public static final String rollerMotorCANBus = "rio";
+
+    static final double gearRatio = 1.5;
+    static final double rollerMOI = 0.04;
```
Comment (Meerlikeskeer) [inline 2025_PreseasonSwerve/src/main/java/frc/robot/subsystems/intake/rollers/RollersConstants.java:9]: Please ask CAD for those values.

File: 2025_PreseasonSwerve/src/main/java/frc/robot/subsystems/intake/rollers/RollersIOSim.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    @Override
+    public void setVoltage(double volts){
+        appliedVoltage = volts;
+        rollerMotorSim.setInputVoltage(volts);
+    }
+   
+        @Override
+        public void setVelocity(double velocity){
+            desiredVelocity = velocity;
+
```
Comment (Meerlikeskeer) [inline 2025_PreseasonSwerve/src/main/java/frc/robot/subsystems/intake/rollers/RollersIOSim.java]: There is no PID here, and the motor isn't assigned a output. Add velocity PID and FF and apply the output to the motor.

**Review by Meerlikeskeer — COMMENTED** [review]
i reviewed and left comments

**Comment by nonchalantnia** [comment]
hi

**Pull request closed without merging**